src/encoded/static/components/item-pages/WorkflowView.js

```diff
@@ -65,7 +65,7 @@ export function commonGraphPropsFromProps(props){
     };
 
     if (props.isFullscreen) {
-        graphProps.width = props.windowWidth;
+        graphProps.width = props.windowWidth - 40;
```

Comment:
There's a 20px padding-left and padding-right on container when in full screen. This accounts for it so we don't have a horizontal scrollbar for the graph container (unless graph width actually exceeds it).

The presence of horizontal scrollbar / graph position on full screen is not 100% consistent at times because the browser seems to return different value for `window.innerWidth` depending on if mouse is attached or not because if mouse is attached, and screen is scrollable, Chrome will add / account-for a vertical scrollbar which is about 15px wide.